package.json

```diff
@@ -21,7 +21,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.2",
-    "@mate-academy/scripts": "^0.9.2",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.